### ISSUE TITLE
[SI-WARNING-TRUE] 1.3.1 Info and Relationships - Non-distinguishable landmarks

### ISSUE THREAD

**Issue by cathysnider** [CuBoulder/express#943]
## Problem Description per Siteimprove

The page contains two or more HTML5 or WAI-ARIA landmarks of the same type that has not been named. Users might not know the difference if it's not somehow explained.

## Page Description

Home page of a completely fresh Web Express install

## What we're doing: 

We've got two regions with attribute role=navigation. 
< nav role="navigation" tabindex="-1" class="region region-menu" style="z-index: 2147483646; position: relative;"> < /nav >
< nav id="mobile-menu" role="navigation">< /nav >

## Solution

These should have aria-labels . Add attribute aria-label="Site menu" and aria-label="Site menu for mobile"  to create unique names that describe the purpose of each landmark.
